scripts/linters/pylint_extensions.py

```diff
@@ -2139,6 +2140,131 @@ def visit_classdef(self, node):
+        ),
+        'C0036': (
+            'Please add schema in HANDLER_ARGS_SCHEMA for %s class. \nVisit '
+            'https://github.com/oppia/oppia/wiki/Validation-of-handler-args'
+            'to learn how to write schema for handlers.',
+            'no-schema-for-handler-args',
+            'Enforce writing schema for request arguments of handler class.'
+        ),
+        'C0037': (
+            'URL_PATH_ARGS_SCHEMAS for %s class must be dict.',
+            'url-path-args-schemas-must-be-dict',
+            'Enforce URL_ARGS_PATH_SCHEMAS must be of dict type.'
+        ),
+        'C0038': (
+            'HANDLER_ARGS_SCHEMAS for %s class must be dict.',
+            'handler-args-schemas-must-be-dict',
+            'Enforce HANDLER_ARGS_SCHEMAS must be of dict type.'
+        )
+    }
+
+    def check_url_path_args_schemas_to_be_a_dict(self, node):
+        """Checks whether URL_PATH_ARGS_SCHEMAS of a class is of dict type.
+
+        Args:
+            node: astroid.nodes.ClassDef. Node for a class definition
+                in the AST.
+
+        Returns:
+            bool. Whether URL_PATH_ARGS_SCHEMAS of a class is of dict type.
+        """
+        if 'URL_PATH_ARGS_SCHEMAS' not in node.locals:
+            return True
+
+        generator_object_for_value_of_schemas = (
+            node.locals['URL_PATH_ARGS_SCHEMAS'][0].assigned_stmts())
+
+        for value_of_schemas in generator_object_for_value_of_schemas:
+            if value_of_schemas.name != 'dict':
+                return False
+        return True
+
+    def check_handler_args_schemas_to_be_a_dict(self, node):
+        """Checks whether HANDLER_ARGS_SCHEMAS of a class is of dict type.
+
+        Args:
+            node: astroid.nodes.ClassDef. Node for a class definition
+                in the AST.
+
+        Returns:
+            bool. Whether HANDLER_ARGS_SCHEMAS of a class is of dict type.
+        """
+        if 'HANDLER_ARGS_SCHEMAS' not in node.locals:
+            return True
+
+        generator_object_for_value_of_schemas = (
+            node.locals['HANDLER_ARGS_SCHEMAS'][0].assigned_stmts())
+
+        for value_of_schemas in generator_object_for_value_of_schemas:
+            if value_of_schemas.name != 'dict':
+                return False
+        return True
+
+    def check_parent_class_is_basehandler(self, node):
+        """Checks whether the parent class of given class is BaseHandler.
+
+        Args:
+            node: astroid.nodes.ClassDef. Node for a class definition
+                in the AST.
+
+        Returns:
+            bool. Whether the parent class of given class is BaseHandler.
+        """
+        parent_class_is_basehandler = False
+        for ancestor_node in node.ancestors():
+            if ancestor_node.name == u'BaseHandler':
+                parent_class_is_basehandler = True
+                break
+        return parent_class_is_basehandler
```

Comment:
You can make it even simpler.
```suggestion
        for ancestor_node in node.ancestors():
            if ancestor_node.name == u'BaseHandler':
                return True
        return False
```

Reply:
Done